apidoc/Facebook.yml

```diff
@@ -661,7 +661,7 @@ methods:
             A dictionary object containing required and optional parameters.
         type: InviteDialogParams
     since: "5.4.0"
-    platforms: [iphone, ipad]
+    platforms: [iphone, ipad, android]
```

Comment:
Feature-freeze is tomorrow, so if you can approve it until tomorrow, we could deliver the new API cross-platform from the beginning. But it needs a FT